src/main/resources/application-standalone.conf

```diff
@@ -48,13 +48,6 @@ authorization-server {
 
 interop-commons {
   isInternetFacing = true
-
-  vault {
-    address = ${VAULT_ADDR}
-    token = ${VAULT_TOKEN}
-    signature-route = ${VAULT_SIGNATURE_ROUTE}
-  }
-
```

Comment:
Shouldn't these be replaced with aws kms settings? How does the app know how to connect to that?

Reply:
The only configuration for KMS is:

interop-commons.kms {
  max-concurrency = 50
}

Reply:
Fine, I didn't remember it, thanks